src/ngx_http_lua_ssl_certby.c

```diff
@@ -298,8 +298,14 @@ ngx_http_lua_ssl_cert_handler(ngx_ssl_conn_t *ssl_conn, void *data)
 
     c->log->action = "loading SSL certificate by lua";
 
-    rc = lscf->srv.ssl_cert_handler(r, lscf, L);
-
+    if (lscf->srv.ssl_cert_handler) {
```

Comment:
I agree, I have to go deeper and find out the real reason of null pointer. 
But, anyway, 
- There is only one 3rd party module I use, besides shipped with nginx and openresty: https://github.com/newobj/nginx-x-rid-header/blob/master/ngx_x_rid_header_module.c, it's quite simple and i'ts not the culprit for my mind. 
- I have used my build of nginx for rather long time, with no segfaults just untill I've added only one line: "ssl_certificate_by_lua_block {  return  }"
- there are no any other block of lua code in my config
- It seems, I've somehow localized this issue,. My nginx crashes after request from IP addresses which use okhttp/3.3.1 client (it's java http client for android)

Some more info I've posted to https://groups.google.com/forum/#!topic/openresty-en/ym1oPD9fyLo